packages/opentelemetry-context-zone-peer-dep/src/util.ts

```diff
@@ -15,13 +15,17 @@
  */
 
 /**
- * check if an object has addEventListener and removeEventListener functions then it will return true.
- * Generally only called with a `TargetWithEvents` but may be called with an unknown / any.
+ * check if an object has `addEventListener` and `removeEventListener` functions.
+ * Generally only called with a `TargetWithEvents` but may be called with an `unknown` value.
  * @param obj - The object to check.
  */
-export function isListenerObject(obj: any = {}): boolean {
```

Comment:
The parameter default seems pointless as that will just always return `false`. Removed it since this is a private utility not a public export.